src/dsp_tools/utils/connection_live.py

```diff
@@ -352,10 +351,10 @@ def _mask(self, sensitive_info: str) -> str:
         else:
             return f"{sensitive_info[:unmasked_until]}[+{len(sensitive_info) - unmasked_until}]"
 
-    def _should_retry(self, response: Response) -> bool:
-        in_500_range = HTTP_SERVER_ERROR_LOWER <= response.status_code <= HTTP_SERVER_ERROR_UPPER
-        try_again_later = "try again later" in response.text
-        return try_again_later or in_500_range
+    def _in_ci(self) -> bool:
+        # https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables
+        ci = os.getenv("GITHUB_ACTIONS")
```

Comment:
if we don't do that, the CI tests run very long, and it takes too much time until we know that a test failed